go.mod

```diff
@@ -5,3 +5,5 @@ go 1.23.3
 require golang.org/x/text v0.21.0
 
 require golang.org/x/net v0.33.0
+
+replace github.com/kashifkhan0771/utils => ./rand
```

Comment:
Why are we doing this? By doing this, Go will consider `./rand` as the root of the `github.com/kashifkhan0771/utils` module. Other packages (string, pointers, maps, etc.) in the library will not be accessible because Go won't look beyond the rand directory.

Reply:
You're totally right, I fixed it in the newest commit